fairscale/experimental/nn/ssd_offload.py

```diff
@@ -21,7 +24,8 @@
     pass
 
 
-DEFAULT_CHUNK_SIZE = 1024 * 1024
+# DEFAULT_CHUNK_SIZE = 1024 * 1024
```

Comment:
remove?

Reply:
Yes I will remove that in the next update to this PR